src/Endpoints/BankAccounts.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace PagarMe\Endpoints;
+
+use PagarMe\Client;
+use PagarMe\Routes;
+use PagarMe\Endpoints\EndpointInterface;
```

Comment:
You don't need to declare this interface here, as `\PagarMe\Endpoints\Endpoint` already implements it :wink: